atlite/cutout.py

```diff
@@ -564,6 +567,27 @@ def intersectionmatrix(self, shapes, shapes_crs=4326):
         """
         return compute_intersectionmatrix(self.grid, shapes, self.crs, shapes_crs)
 
+    def area(self, crs=3035):
```

Comment:
I don't like the default `crs=3035` - it's only accurate for Europe.

By default I think it should use the CRS of the cutout or throw an error (asking the user to specify a CRS).

Reply:
This actually a good point, thanks for raising. I adjusted the code accordingly.